service/src/main/resources/application.yml

```diff
@@ -3,6 +3,9 @@ server:
 
 spring:
   application.name: terra-azure-relay-listener
+  cache:
+    cache-names: expiresAt
+    caffeine.spec: maximumSize=100,expireAfterAccess=3600s
```

Comment:
Is the token expiration rolling (the same token can be renewed/extended when calling the google API) or fixed (user must get another token when one expires)? If the former, then we should adjust the expiration to be equal, or slightly less than the max duration of the token.  Otherwise, tokens will be considered expired when in fact they are not.

Reply:
it's fixed...Terra UI will peridically keep the token refreshed I think